pallets/communities/src/lib.rs

```diff
@@ -0,0 +1,140 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+// #[cfg(feature = "runtime-benchmarks")]
+// mod benchmarking;
+
+use frame_support::RuntimeDebug;
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, MaxEncodedLen, RuntimeDebug, TypeInfo)]
+#[scale_info(skip_type_params(T))]
+#[codec(mel_bound(T: pallet::Config))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Community<T: pallet::Config> {
+	// TODO : Maybe every community can configure an asset?
+	pub controller: T::AccountId,
+	pub population: PopulationOf<T>,
```

Comment:
I think we can keep track of population as a simple counter just for reference, it's not good to have the accounts here as it restricts how many users a business can have or how many people are supposed to live in a place. Instead we can do the inverse and track in the user account data which community/marketplaces they belong to